fdbserver/worker.actor.cpp

```diff
@@ -3314,7 +3314,9 @@ ACTOR Future<Void> fdbd(Reference<IClusterConnectionRecord> connRecord,
 		    makeReference<AsyncVar<ClusterControllerPriorityInfo>>(getCCPriorityInfo(fitnessFilePath, processClass));
 		auto serverDBInfo = ServerDBInfo();
 		serverDBInfo.client.isEncryptionEnabled = SERVER_KNOBS->ENABLE_ENCRYPTION;
+		serverDBInfo.myLocality = localities;
 		auto dbInfo = makeReference<AsyncVar<ServerDBInfo>>(serverDBInfo);
+		TraceEvent("MyLocality").detail("Locality", dbInfo->get().myLocality.toString());
```

Comment:
this seems to be a little different from what we have in our test [here](https://github.com/apple/foundationdb/pull/7962/files#diff-bacdd17dcf02addd67bd0c0c8293ab3c38b13d76de1e1ddb748eec9fb9d5c02dR2925-R2929), looks like it is okay.

Reply:
oh because we were on 7.1, let me test it again anyways

Reply:
Yes, this is more efficient.